packages/system/keycloak-configure/templates/configure-kk.yaml

```
@@ -1,9 +1,61 @@
+{{- if $existingK8sSecret }}
+  {{- $k8sClient = index $existingK8sSecret.data "client-secret-key" | b64dec }}
+{{- end }}
+
+{{ $kubeappsClient := randAlphaNum 32 }}
+{{- if $existingKubeappsSecret }}
+  {{- $kubeappsClient = index $existingKubeappsSecret.data "client-secret-key" | b64dec }}
+{{- end }}
+
+{{ $cookieSecret := randAlphaNum 16 }}
+{{- if $existingAuthConfig }}
+  {{- $cookieSecret = index $existingAuthConfig.data "cookieSecret" | b64dec }}
```

yamllint (1.35.1)

[warning] wrong indentation: expected 0 but found 2

(indentation)